dockerfiles/che-fabric8/Dockerfile

```diff
@@ -43,6 +43,22 @@ RUN mkdir -p      /etc/pcp /var/run/pcp /var/lib/pcp /var/log/pcp
 RUN chmod -R 0777 /etc/pcp /var/run/pcp /var/lib/pcp /var/log/pcp
 EXPOSE 44321
 
+
+# Agent bond including Jolokia and jmx_exporter
```

Comment:
Since there is a centos image with JDK 8 and `Agent Bond` [here](https://github.com/fabric8io-images/java/tree/master/images/centos/openjdk8/jdk) I think we should use it instead of manually adding all of this into RhChe repo.

Reply:
@ibuziuk @davidfestal @l0rd @amisevsk WDYT?

Reply:
The question is: is there the equivalent for RHEL. We now manage 2 versions of the DockerFile.

Reply:
BTW @skabashnyuk Do you plan to do the same changes in Dockerfile.rhel ?
That would be needed to have JMX on prod and prod-preview I assume, no ?

Reply:
ur right, there is no rhel version. Maybe we should ask to provide one